crates/ruma-client-api/src/r0/uiaa.rs

```diff
@@ -433,3 +433,5 @@ mod tests {
         );
     }
 }
+
+pub mod fallback;
```

Comment:
I think we have all our module declarations near the top of the file in the rest of the code.